setup.py

```diff
@@ -60,7 +60,7 @@
 Topic :: Software Development :: Libraries :: Python Modules
 """.splitlines())))
 
-INSTALL_REQUIRES = ['pyperclip >= 1.5.27', 'colorama']
+INSTALL_REQUIRES = ['pyperclip >= 1.5.27', 'colorama', 'attrs']
```

Comment:
Anytime we add a dependency we should updated the README.md accordingly since we have a section which discusses the required dependencies.